build-tools/script/diff2html.js

```diff
@@ -0,0 +1,61 @@
+const { exec } = require('child_process');
+const fs = require('fs');
+const inquirer = require('inquirer');
+
+const OUTPUT_DIR = 'dist/diff';
+const REPORT_NAME = 'templates.html';
+const EXT = '*.hbs';
+
+const questions = [
+  {
+    type: 'list',
+    name: 'type',
+    message:
+      'The script will generate diff reports for handlebars templates.\n What do you want to use as a diff to the current commit?',
+    choices: [
+      {
+        name: 'Commit hash',
+        value: 'hash',
+      },
+      {
+        name: 'Master branch',
+        value: 'master',
+      },
+    ],
+  },
+  {
+    type: 'input',
+    name: 'commitHash',
+    message: 'Please enter a commit hash',
+    when: answers => answers.type === 'hash',
+    validate: val => val.length > 0,
+  },
+];
+
+const createDir = () => {
```

Comment:
Could we use [fs-extra](https://github.com/jprichardson/node-fs-extra) for this? Has methods to easily create files/folders without having to check.